tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -528,6 +528,106 @@ void test_suite_for_map_on_sphere(
   test_helper(map2);
 }
 
+/*!
+ * \ingroup TestingFrameworkGroup
+ * \brief Given a Map `map`, tests the map functions, including map inverse,
+ * jacobian, and inverse jacobian, for a series of points.
+ * These points are chosen in a right cylinder with cylindrical radius 1
+ * and I1-axis extending from -1 to +1. The
+ * map is expected to be valid on the boundary of that cylinder as well as
+ * in its interior.
+ * This test works only in 3 dimensions.
+ */
+template <typename Map>
+void test_suite_for_map_on_cylinder(const Map& map,
+                                    const double inner_radius,
+                                    const double outer_radius) noexcept {
+  static_assert(Map::dim == 3, "Works only for a 3d map");
```

Comment:
Expand error message to say "test_suite_for_map_on_cylinder only works for a 3D map"